DEVELOPMENT.md

```diff
@@ -0,0 +1,219 @@
+# Development
+
+## Prerequisites
+
+* git
+* IntelliJ 2020.1 Community Edition/Ultimate
```

Comment:
To avoid constantly bumping this version as well when a new release comes out: just
```suggestion
* latest IntelliJ Community Edition/Ultimate
```

Reply:
@micpiotrowski: doesn't seem updated

Reply:
Ouh, my fault, done.